packages/runtime/container-runtime/src/test/getPendingBlobs.spec.ts

```diff
@@ -42,7 +42,6 @@ describe("getPendingLocalState", () => {
 
 		const runtime2 = new MockRuntime(mc, summaryData, false, pendingState);
 		await runtime2.attach();
-		await runtime2.processStashed();
```

Comment:
why is this safe to remove, doesn't it change the result of the test?

Reply:
Now it's inside connect()

Reply:
It's closer to real scenario after https://github.com/microsoft/FluidFramework/pull/17417